apps/api/src/domain/models/audio-visual/audio-item/repositories/arango-audio-item-query-repository.spec.ts

```diff
@@ -366,4 +367,49 @@ describe(`ArangoAudioItemQueryRepository`, () => {
             expect(numberOfItems).toBe(1);
         });
     });
+
+    describe(`import line items`, () => {
+        const participant = new TranscriptParticipant({
+            initials: 'WO',
+            name: 'William Who',
+        });
+
+        const speakerInitials = participant.initials;
+
+        const targetAudioItem = buildTestInstance(EventSourcedAudioItemViewModel, {
+            transcript: Transcript.buildEmpty(),
+        });
+
+        const lineItems = [1, 2, 3, 4, 5].map((i) =>
+            buildTestInstance(TranscriptLineItemDto, {
+                inPointMilliseconds: 100 * i,
+                outPointMilliseconds: 100 * i + 50,
+                speakerInitials,
+                text: `text for line item #${i}`,
+            })
+        );
+
+        beforeEach(async () => {
+            await testQueryRepository.create(targetAudioItem);
+        });
+
+        it('should have a test', async () => {
+            await testQueryRepository.importLineItems(targetAudioItem.id, lineItems);
+
+            const updatedView = (await testQueryRepository.fetchById(
+                targetAudioItem.id
+            )) as EventSourcedAudioItemViewModel;
+
+            const numberOfItems = updatedView.transcript.countLineItems();
+
+            expect(numberOfItems).toBe(lineItems.length);
+
+            const missingLineItems = lineItems.filter(
+                ({ inPointMilliseconds, outPointMilliseconds }) =>
+                    !updatedView.transcript.hasLineItem(inPointMilliseconds, outPointMilliseconds)
+            );
+
+            expect(missingLineItems).toEqual([]);
```

Comment:
Since we have a detailed assertion here, I wonder if we can stick to a sanity check (by count) in the handler. I'm struggling with whose responsibility it is to check in detail or whether both should.